### DISCUSSION TITLE
On Android, deleting a local file, should delete it from ente

### DISCUSSION THREAD

**Discussion by TheAmogh** [ente/ente#3546] [Enhancements]
I'm sure this has been requested in many different, often conflicting ways. I don't know what is the right approach.

I'm on Android. I sometimes use a different gallery app or another file manager to view my photos, and occasionally delete the ones I don't want stored anywhere. The current behavious of Ente is to keep the photos in cloud regardless of whether it is on device or not. This obviously makes sense since you seem to have a cloud first approach, and also that you have a "Free up space" which precisely removes photos on device if they're on cloud.

So my ask might be against the grain, but I think Google Photos does what I expect. I delete photos locally, it gets removed from the Google cloud.
Is there a way to please both crowds: ones who want cloud to be in sync with the device (Device -> Cloud sync) & ones who want to have a cloud first experience (Everything in cloud, pull to device as required)

**Comment by rbxrt** [comment]
I would be very happy about that too. It's one of the missing features that is currently holding me back from switching completely to ente, because I'm afraid it will quickly end up in a mess.

**Comment by rokfor2000** [comment]
I've read in another thread that "[When you open the device folder, you should still see the message that some of the items are ignored for delete because they were previously deleted](https://github.com/ente-io/ente/discussions/3242#discussioncomment-10622621).". (The other one is iOS, hence writing here) I seem to have exactly the same issue. Once the photos upload and I delete them via web interface, as soon as I use app on my phone they all resync. 

Any advice would be great, thanks.

**Reply by vishnukvmd** [reply]
Deleted items from the web interface should not be resyncing. If this happens again, could you please share logs from mobile with support@ente.io? Thanks.

**Comment by wabunting** [comment]
A related feature that would help with this is a “not on device” icon much like the currently existing “not in cloud” icon in the bottom left corner of a photo preview. This would at least let the user quickly clean up anything they have already deleted from their device, especially if there are added filter options for on device/not on device and in cloud/not in cloud.

**Reply by gexcube** [reply]
This!

**Comment by SanskritFritz** [comment]
This is the single issue that I too would loooooove in Ente. Please, make this a reality.

**Comment by gexcube** [comment]
+1, or at least the filter "not on device".

**Reply by SanskritFritz** [reply]
> +1, or at least the filter "not on device".

Yes, good point, that would be even better!

**Comment by gothman5** [comment]
Yes please! I don't want to go through the effort or deleting photos twice- once on my device and Google photos (default gallery for Android on Google Pixel phone) and then again on Ente 🙈

**Reply by Xaviber76** [reply]
Although I understand the annoyance of deleting photos twice, I don't think Ente has away to know why or how you deleted your pictures. For example, if deleting a picture from your device would delete it too from Ente, when you free up space from Google Photos, Ente would delete all that pictures because they are not on your device anymore.

**Reply by gexcube** [reply]
@Xaviber76 I understand that might be challenge. That is why I suggest as first step to just implement filter to display "on cloud only", "on device + on cloud", "on device only". This way one can easily do that. 

Today I have to differentiate by pressing delete button and seeing different question being asked. This way I see what picture is on cloud only. One per one image, quite annoying. Filter/icon could solve most of the issue by allow bulk cleanup once in a while.
